### PR TITLE
Implement Security Misconfiguration Vulnerability

### PR DESCRIPTION
**Pull request by nstapler**
### Major

- Add pool connections to different db roles with appropriate querying privileges

- New setenvs file(pinned in slack)
### Minor

- Enable account page to display `N/A` for personal info when empty

### CODE REVIEW

File: Vulnerabilities/securitymisconfiguration.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# [securitymisconfiguration](https://owasp.org/www-project-top-ten/2017/A6_2017-Security_Misconfiguration)
```
Comment (martinky24) [inline Vulnerabilities/securitymisconfiguration.md:1]: `securitymisconfiguration` => `Security Misconfiguration` ? (seems consistent with what we have elsewhere)

File: Vulnerabilities/securitymisconfiguration.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Security Misconfiguration is when application settings, tools, or services allow for unintended access or functionality. This could be returning server error stack
+traces to the client or the default configurations for a certain framework allowing all users admin-like usage.
+## Our Implementation
+
+In our app, we demonstrate a case of injection by not properly parameterizing the "transfers" portion of our site. In both the insecure and secure versions of our website, a destination account number and amount is prompted from the user. However, in the insecure version of our website, the user can enter any string into the account number input, and the server does not perform checks to ensure that the account number is numerical. This allows the user to manipulate the transfer query directly.
```
Comment (martinky24) [inline Vulnerabilities/securitymisconfiguration.md:7]: `we demonstrate a case of injection` typo?

File: Website/routes/login.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ var loginUser = async function loginUserAction(req, res, next){
 	if (result.rows.length > 0 && bcrypt.compareSync(password, result.rows[0].password_hash)) {
 		req.session.uname = username;
 		req.session.userID = result.rows[0].user_id // Store for easy personal/financial lookup
-		console.log('Insecurely logging in as user ',req.session.uname, "\nWith uid:", req.session.userID);
+		console.log('Logging in as user ',req.session.uname, "\nWith uid:", req.session.userID);
```
Comment (martinky24) [inline Website/routes/login.js:33]: very minor bugfix I just added here as it should be entirely inconsequential

**Review by martinky24 — COMMENTED** [review]
You'll want to link your markdown file from the homepage!

File: Vulnerabilities/securitymisconfiguration.md (hunk 1 above)
Comment (nstapler) [reply Vulnerabilities/securitymisconfiguration.md:1]: Lol good catch

File: Vulnerabilities/securitymisconfiguration.md (hunk 2 above)
Comment (nstapler) [reply Vulnerabilities/securitymisconfiguration.md:7]: ah, i skipped over that section when i copied->then edited

**Review by martinky24 — APPROVED** [review]

**Review by tbetley — APPROVED** [review]
Easy implementation, looks good.

**Pull request merged**